modular/downloader/download_task.go

```diff
@@ -279,17 +279,6 @@ func (d *DownloadModular) PreDownloadPiece(ctx context.Context, downloadPieceTas
 		bucketID := downloadPieceTask.GetBucketInfo().Id.Uint64()
 		bucketName := downloadPieceTask.GetBucketInfo().GetBucketName()
 
-		// if stream account has been frozen, it is not allowed to download
```

Comment:
Is the below logic no longer valid?

Reply:
no, if it is getObject handler , it is already checked in func (a *AuthenticationModular) VerifyAuthentication() . So This is a duplicate check